.github/workflows/frontend-test.yml

```diff
@@ -7,22 +7,44 @@ defaults:
     shell: bash -l {0}
 
 jobs:
-  # This workflow contains a single job called "build"
-  build:
-    # The type of runner that the job will run on
-    runs-on: ubuntu-latest
+  run-frontend-tests:
+    name: Run Frontend Tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      max-parallel: 4
+      matrix:
+        os: [ubuntu-latest]
 
-    # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
-      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
-      - uses: actions/checkout@v3
+      - name: Check out repo
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0 # Fetch complete history for accurate versioning
```

Comment:
Ahh thanks, I guess I never really knew what this was for. I'll remove it because so far the scope of this workflow is just running frontend tests anyway